qa/workunits/cephadm/test_iscsi_etc_hosts.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# checks if the container and host's /etc/hosts files match
+# Necessary to avoid potential bugs caused by podman making
+# edits to /etc/hosts file in the container
+# exits with code 1 if host and iscsi container /etc/hosts do no match
+
+set -e
+set -x
+FSID=$(/home/ubuntu/cephtest/cephadm shell -- ceph fsid)
```

Comment:
This variable is unused.

Reply:
oops, left over from the docker vs. podman check. Removed it.